patronload/config.py

```diff
@@ -83,15 +83,11 @@ def configure_sentry() -> str:
 def load_config_values() -> dict:
     """Retrieve all required env variables to update the config_values dict."""
     config_values = {
-        "DATA_WAREHOUSE_USER": "user123",
-        "DATA_WAREHOUSE_PASSWORD": "pass123",
-        "DATA_WAREHOUSE_HOST": "http://localhost",
-        "DATA_WAREHOUSE_PORT": "1234",
-        "DATA_WAREHOUSE_SID": "database5678",
         "S3_BUCKET_NAME": "patronload",
```

Comment:
Why not just use a list for the config value keys? It doesn't like the default values are being used.

Reply:
👍